posthog/hogql_queries/insights/trends/breakdown_values.py

```diff
@@ -0,0 +1,81 @@
+from typing import List
+from posthog.hogql import ast
+from posthog.hogql.parser import parse_expr, parse_select
+from posthog.hogql.query import execute_hogql_query
+from posthog.hogql_queries.utils.query_date_range import QueryDateRange
+from posthog.models.team.team import Team
+
+
+class BreakdownValues:
+    team: Team
+    event_name: str
+    breakdown_field: str
+    query_date_range: QueryDateRange
+
+    def __init__(self, team: Team, event_name: str, breakdown_field: str, query_date_range: QueryDateRange):
+        self.team = team
+        self.event_name = event_name
+        self.breakdown_field = breakdown_field
+        self.query_date_range = query_date_range
+
+    def get_breakdown_values(self) -> List[str]:
+        select_field = ast.Alias(alias="value", expr=ast.Field(chain=["properties", self.breakdown_field]))
+
+        query = parse_select(
+            """
+                SELECT groupArray(value) FROM (
+                    SELECT
+                        {select_field},
+                        count(*) as count
+                    FROM
+                        events e
+                    WHERE
+                        {events_where}
+                    GROUP BY
+                        value
+                    ORDER BY
+                        count DESC,
+                        value DESC
+                    LIMIT 25
+                    OFFSET 0
+                )
+            """,
+            placeholders={
+                "events_where": self._where_filter(),
+                "team_id": ast.Constant(value=self.team.pk),
```

Comment:
We don't need the `team_id` field in HogQL, it gets added automatically. Seems like this placeholder is not in use here anyway.

Reply:
Ahhh yes, unsure why this is here - but v clever that it gets added automatically. Does this apply to every query on `events`?